### PR TITLE
Update support page content

### PR DESCRIPTION
**Pull request by saimaghafoor**
Updated content for support page so that it's clear what our support offer is.

### CODE REVIEW

File: app/templates/views/support/index.html [hunk 1]
```diff
@@ -22,16 +24,8 @@ <h1 class="heading-large">Support</h1>
...
-  <div class="govuk-grid-column-two-thirds">
-    <h2 class="heading-medium">Office hours</h2>
-    <p class="govuk-body">Our office hours are 9:30am to 5:30pm, Monday to Friday.</p>
-    <p class="govuk-body">When you report a problem in office hours, we’ll aim to read it within 30 minutes and reply within one working day.</p>
 
-    <h2 class="heading-medium">Out-of-hours</h2>
-    <p class="govuk-body">Outside office hours, response times depend on whether you’re reporting an emergency.</p>
-    <p class="govuk-body">If it’s an emergency, we’ll reply within 30 minutes and update you every hour until the problem’s fixed.</p>
-    <p class="govuk-body">If your problem is not an emergency, we’ll reply within one working day.</p>
+    <h2 class="heading-medium">What counts as an emergency?</h2>
```
Comment (karlchillmaid) [inline app/templates/views/support/index.html:28]: I think we need to keep lines 25 and 26, as they define the layout of this next chunk of content.

File: app/templates/views/support/public.html [hunk 2]
```diff
@@ -26,8 +26,6 @@
       There are other pages on GOV.UK where you can get help:
     </p>
 
-    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">
-      <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/coronavirus">Coronavirus (COVID-19)</a>
     </h2>
```
Comment (karlchillmaid) [inline app/templates/views/support/public.html:29]: You’ll also need to delete lines 29 to 32:

`</h2>`
`<p class="govuk-body">`
`  Find guidance and support.`
`</p>

**Comment by karlchillmaid** [comment]
I suggest checking if we need to update any of the content in these files too:

* `[bat-phone.html](https://github.com/alphagov/notifications-admin/blob/main/app/templates/views/support/bat-phone.html)`
* `[thanks.html](https://github.com/alphagov/notifications-admin/blob/main/app/templates/views/support/thanks.html)`
* `[triage.html](https://github.com/alphagov/notifications-admin/blob/main/app/templates/views/support/triage.html)`

**Comment by saimaghafoor** [comment]
Thanks @karlchillmaid. Not too sure where these pages are (front end) so would be great for some support on this as wouldn't want t break anything by changing them!

File: app/templates/views/support/bat-phone.html [hunk 3]
```diff
@@ -17,26 +17,31 @@
...
+          We’ll reply within 30 minutes and update you every hour until we fix the problem.
         </p>
-        <h2 class="heading-medium">Any other problems</h2>
+
+        <h2 class="heading-medium">If it’s not an emergency</h2>
         <p class="bottom-gutter-2">
           <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.feedback', ticket_type='report-problem', severe='no') }}">Fill in this form</a>
-          and we’ll get back to you by the next working day.
+          and we’ll reply within 1 working day.
         </p>
```
Comment (karlchillmaid) [inline app/templates/views/support/bat-phone.html:45]: Let’s discuss whether we need this here.

**Comment by karlchillmaid** [comment]
# To do

@saimaghafoor 

1. Change the h1 on `triage.html` to ‘Is it an emergency?’ This will also mean updating the form so we don’t repeat the question twice. I’d suggest asking a developer to pair on making this change.

1. Pair with a developer to fix whatever is breaking the checks on this PR (probably some of the automated tests)

File: app/templates/views/support/bat-phone.html [hunk 4]
```diff
@@ -3,7 +3,7 @@
 {% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
 
 {% block per_page_title %}
-  Out of hours emergencies
+  Report an out of hours emergency
```
Comment (karlchillmaid) [inline app/templates/views/support/bat-phone.html:6]:
```suggestion
  Report an emergency out of hours
```

**Comment by saimaghafoor** [comment]
@samuelhwilliams There are other messages on the thanks.html but I'm unsure what situations they appear (messages 3, 4 and 5). Any clarity/help would be amazing so I can amend.

**Review by samuelhwilliams — APPROVED** [review]
🎉 nice

**Pull request merged**